sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -67,8 +67,8 @@
     <spotbugs.excludeFilterFile>spotbugs-exclude.xml</spotbugs.excludeFilterFile>
     <spotless.skip>false</spotless.skip>
 
-    <netty.version>4.1.101.Final</netty.version> <!-- {x-version-update;io.netty:netty-common;external_dependency} -->
-    <netty-tcnative.version>2.0.62.Final</netty-tcnative.version> <!-- {x-version-update;io.netty:netty-tcnative-boringssl-static;external_dependency} -->
+    <netty.version>4.1.108.Final</netty.version> <!-- {x-version-update;io.netty:netty-common;external_dependency} -->
```

Comment:
Should we update the python script to automate updating this version?

Reply:
I think the test should be sufficient as that'll block this issue. I think there would need to be a larger update to the Python tooling to handle this